apps/CommonApp/Gemfile

```diff
@@ -0,0 +1,16 @@
+source 'https://rubygems.org'
```

Comment:
I think most of the config files (ruby, metro, app.json, ...) aren't necessary here as this app won't be built by itself, no?

Reply:
Probably not, main point of this PR was to add shared source for examples and deal with configs in next PRs. However I see that it makes sense to do this right away 😅

Reply:
Done in [ba4ad91](https://github.com/software-mansion/react-native-gesture-handler/pull/3496/commits/ba4ad91a31d918543d7a7b0a6f82e88673ed063e)